admin_manual/maintenance/upgrade.rst

```diff
@@ -91,3 +91,22 @@ Change ``"maintenance" => false`` to ``"maintenance" => true``:
     "maintenance" => true,
 
 Then change it back to ``false`` when you are finished.
+
+Manual steps during upgrade
+---------------------------
+
+Some operation can be quite time consuming. Therefore we decided not to add them
+the the normal upgrade process. We recommend to run them manually after the upgrade
+was completed. Below you find a list of this commands.
+
+Upgrading to Nextcloud 13
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+With Nextcloud 13 we added a new index to the share table which should result in
+significant performance improvements:
```

Comment:
double colon `::` please so that the command call is a code block

Reply:
Here we also don't use `::`` https://github.com/nextcloud/documentation/pull/671/files#diff-368ad62123233efc28400a735b869cf0L82
indention should be enough, preview looks fine too.

Reply:
The colon is in line 80 (last none code line).
Fix is in https://github.com/nextcloud/documentation/pull/676

Not sure which preview you looked at, but see the screenshots in my PR